### PR TITLE
chore: embed-services package

### PR DESCRIPTION
**Pull request by josephaxisa**
This is just a skeleton of the package that will contain the service layer for looker embed components.

### CODE REVIEW

File: packages/embed-services/package.json
Comment (jkaster) [inline packages/embed-services/package.json:14]: I think you want this private for now

File: packages/embed-services/src/index.ts
Comment (jkaster) [reply packages/embed-services/src/index.ts:5]: Yes, just let the linter do its thing. It's based on the `LICENSE` from the main folder, which should be updated to 2023, then your local `LICENSE` file matches.

**Review by jkaster — CHANGES_REQUESTED** [review]
You should update the main repo LICENSE file for the current copyright year

File: packages/embed-services/package.json
Comment (jkaster) [inline packages/embed-services/package.json:12]: should this be `true` or `false`?

**Review by jkaster — COMMENTED** [review]
got a question before approval

File: packages/embed-services/package.json
Comment (josephaxisa) [reply packages/embed-services/package.json]: good catch, thanks.

**Review by jkaster — APPROVED** [review]

**Pull request merged**
